Common/include/config_structure.hpp

```diff
@@ -5109,6 +5110,18 @@ class CConfig {
    */
   string GetMultizone_HistoryFileName(string val_filename, int val_iZone);
   
+  /*!
+   * \brief Append the zone index to the restart or the solution files.
```

Comment:
Please update these comments.. also MultiInstance with the capitalization of the second "i" might be better for readability.

Reply:
Done. I had left it as Multiinstance in analogy to Multizone, but I agree it improves readibility.
Anyway, I think this should go with a potential new output structure.